### ISSUE TITLE
KeepAlive doesn't seem to be working, 2019-06-09

### ISSUE THREAD

**Issue by tfabris** [tfabris/CrowCam#39]
Nothing unusual is in the logs, but the DVR feed is jumping around, and the times where it jumps seems to be related to when I logged in to watch the stream. In other words, the gaps in the DVR stream are in areas before I watched the stream, and then it starts DVR'ing again at the moments I watched the stream. Two jumps I know about in the DVR stream are:

2019-06-09 - Jump from 1:53pm (about 30-40 mins after the midday bounce) to 3:29 pm (a time when I tried to log in and look at the stream on YouTube). 
2019-06-09 - Jump from 3:55 pm (about 30 mins after that last look) to 4:10pm (when I logged on again).

I won't know until I review the full archive tomorrow, but I suspect the archive will look fine and only the DVR will have had the jumps in it.

I think that YouTube-DL might not be working correctly again. Check it when you get a chance.

**Comment by tfabris** [comment]
On the morning of 2019-06-10 it all seemed to be working fine, no repeat of the problem. Stream DVR history was visible from 5am to 10am (more than four hours!).

Checked YouTube-DL versioning, and its last release on its download page was 2019-06-08, though I can't tell what time of day or what timezone. This indicates to me that, unless their actual file releases lag behind their versioning, then this wasn't a case of YouTube-DL malfunctioning and needing to be updated.

To do: 
- Double check Synology logs for any further indications.
- If nothing seems amiss, then close this bug as "Cannot reproduce".

**Comment by tfabris** [comment]
Looked at the logs again (attached). I see:
- Network issues that day, but not at the time of day that I observed the stream DVR skip.
- YouTube API fails to respond correctly at a certain point, but not at the same time of day either.
- No complaints by YouTube-DL except once at 5am when it failed to download an update of YouTube-DL.

Closing this bug as "cannot reproduce".

Logs:
[2019-06-09 Logs.txt](https://github.com/tfabris/CrowCam/files/3274699/2019-06-09.Logs.txt)
